src/Course/FileIO.hs

```diff
@@ -88,19 +88,21 @@ printFile path contents =
 -- Given a list of (file name and file contents), print each.
 -- Use @printFile@.
 printFiles :: List (FilePath, Chars) -> IO ()
-printFiles files = execIoNoResult printTuple files
+printFiles files = (void . sequence) (printTuple <$> files)
```

Comment:
yeah my apologies for this... you extracted it correctly, and I gave you the wrong feedback about using `void . sequence` instead of creating an abstraction.